ci/rpm7/build.sh

```diff
@@ -233,13 +224,6 @@ echo "===================================== PREPARE ============================
 echo "Builder: create temp folders"
 rm -Rf /tmp/builder || true && mkdir -p /tmp/builder/{db1,db2,db,bu}
 
-if [ -n "${database}" ]; then
-    echo "Builder: starting Mongo"
-    mongod --dbpath /tmp/builder/db  --nojournal --quiet > /dev/null 2>&1 &
-    sleep 3
-    export MONGO_HOST=localhost
```

Comment:
By default, CB uses MongoDB running in localhost (i.e. running in the same container). Now that mongo runs "externaly" to the container as a service (as discussed in [this comment](https://github.com/telefonicaid/fiware-orion/pull/3747/files#r553798313), how do you make CB to use it instead of trying to connect locally?

Reply:
The docker container running the test is connected to the host-network(see the "--network host" option) and therefore the service container is available at localhost inside the container.

Reply:
So you mean that using `--network host` the CB process running inside the container "sees" a MongoDB instance running in localhost at port 27017 even when that MongoDB instance is running *externally* to the container?

Reply:
Yes. We map the service-containers port 27017 to the docker-hosts port 27017 here: https://github.com/wistefan/fiware-orion/blob/master/.github/workflows/functional.yml#L19-L20
And with --network host , the container is attached to the docker-hosts network and sees everything at localhost, that the docker host does.

Reply:
I wonder what would happen if both the CB container and service container start a process listening in the same port (because in that case, "localhost:PORT" would bind to both the container and the service container and that would be a conflict :), but it's fine (we don't have that situation here).

NTC